force-app/main/selector/DonationHistorySelector.cls

```diff
@@ -125,9 +125,12 @@ public inherited sharing class DonationHistorySelector {
         return totalRecords;
     }
 
+
+
     /** 
     * @author Salesforce.org
     * @date 2021
+
```

Comment:
Extra line here, also we normally only include the "author" and "date" at the top of Classes.